skimage/restoration/_denoise_cy.pyx

```diff
@@ -111,22 +73,18 @@ def _denoise_bilateral(image, Py_ssize_t win_size, sigma_color,
                          "`edge`, `wrap`, `symmetric` or `reflect`.")
     cdef char cmode = ord(mode[0].upper())
 
-    cimage = np.ascontiguousarray(image)
 
-    out = np.zeros((rows, cols, dims), dtype=np.double)
-    color_lut = _compute_color_lut(bins, csigma_color, max_value)
-    range_lut = _compute_range_lut(win_size, sigma_spatial)
```

Comment:
We need to call `_compute_range_lut`, which is never done in the new implementation. Rather it is replaced by empty.

Reply:
I personally dislike cython code. Feel free to move this to Python.